packages/synapse-react-client/stories/OrientationBanner.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from '@storybook/react'
+import React from 'react'
+import OrientationBanner, {
+  getOrientationBannerKey,
+  OrientationBannerProps,
+} from '../src/components/OrientationBanner/OrientationBanner'
+
+const meta = {
+  title: 'UI/OrientationBanner',
+  component: OrientationBanner,
+  render: args => {
+    localStorage.removeItem(getOrientationBannerKey(args.name))
```

Comment:
Ensure that if the OrientationBanner is dismissed in the story, the banner will be shown the next time the Story is rendered